src/pages/LogView/LoadingPage/index.tsx

```diff
@@ -46,6 +54,12 @@ const LoadingPage: React.FC<LoadingPageProps> = ({ logType }) => {
         url = getResmokeLogURL(buildID, { raw: true });
         htmlLogURL = getResmokeLogURL(buildID, { html: true });
       }
+      if (logkeeperMetadata) {
+        jobLogsURL = getSpruceJobLogsURL(
+          logkeeperMetadata.task_id,
+          String(logkeeperMetadata.execution || 0)
+        );
```

Comment:
maybe we should allow `execution` in this function to be either a string or a number!